_observing-your-data/prometheusmetrics.md

```diff
@@ -0,0 +1,116 @@
+nav_order: 40
+redirect_from:
+  - /observing-your-data/prometheusmetrics/
+---
+
+# Using Prometheus to visualize metrics
+Introduced 2.4
+{: .label .label-purple }
+
+Starting with OpenSearch 2.4, you can visualize metrics data from the open-source [Prometheus](https://prometheus.io/) monitoring solution as well as from log data aggregated within OpenSearch, unlocking new observability capabilities that allow you to analyze and correlate data across logs and traces as well as metrics. Previously, the OpenSearch observability plugin only allowed you to ingest and visualize logs and traces from your monitored environments. With this feature, you can observe your digital assets with more granularity, gain deeper insight into the health of your infrastructure, and better inform root cause analysis.
+
+![Prometheus and Metrics]({{site.url}}{{site.baseurl}}/images/metrics/metricsgif.gif)
+
+## Creating a connection to Prometheus
+
+You can create a connection from [Prometheus](https://prometheus.io/) to OpenSearch using the SQL plugin in order to view metrics collected from Prometheus in OpenSearch Dashboards.
+
+![Prometheus example 1]({{site.url}}{{site.baseurl}}/images/metrics/metrics4.png)
+
+Upon selection of a metric, the **Metrics explorer** will create a visualization for the selected metric to be displayed, as seen in the following image.
+
+![Prometheus example 2]({{site.url}}{{site.baseurl}}/images/metrics/metrics5.png)
+
+## Configuring a Prometheus data source connection to OpenSearch
+
+To configure a connection to Prometheus for metrics in OpenSearch Dashboards, create a file on your OpenSearch nodes named `datasources.json` containing the Prometheus data source settings. The following examples demonstrate the various Prometheus data source configurations using different authentication methods:
+
+No authentication:
+
+```json
+[{
+    "name" : "my_prometheus",
+    "connector": "prometheus",
+    "properties" : {
+        "prometheus.uri" : "http://localhost:9090"
+    }
+}]
+```
+
+Basic authentication:
+
+```json
+[{
+    "name" : "my_prometheus",
+    "connector": "prometheus",
+    "properties" : {
+        "prometheus.uri" : "http://localhost:9090",
+        "prometheus.auth.type" : "basicauth",
+        "prometheus.auth.username" : "admin",
+        "prometheus.auth.password" : "admin"
+    }
+}]
+```
+
+AWSSigV4 authentication:
+
+```json
+[{
+    "name" : "my_prometheus",
+    "connector": "prometheus",
+    "properties" : {
+        "prometheus.uri" : "http://localhost:8080",
+        "prometheus.auth.type" : "awssigv4",
+        "prometheus.auth.region" : "us-east-1",
+        "prometheus.auth.access_key" : "{{accessKey}}"
+        "prometheus.auth.secret_key" : "{{secretKey}}"
+    }
+}]
+```
+
+After configuring Prometheus in the `datasources.json` file, run the following command to load the configuration into the OpenSearch keystore. The configuration is securely stored as is contains credential information.
```

Comment:
The configuration is securely stored **as is contains** credential information.

I'm not figuring this out.

Reply:
reworded for clarity